utils/tests/test_unit_sample_data_generators.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from utils.sample_data_generators import generate_too_many_digits
+from utils.sample_data_generators import generate_too_many_decimal_places
+
+
+class DecimalDataGenerators(TestCase):
+    def test_too_many_digits_generator_should_return_decimal_with_more_digits_than_declared(self):
+        declared_digits = 4
+        result = generate_too_many_digits(declared_digits)
+        digits = len(str(result).replace('.', ''))
+        self.assertEqual(digits, 5)
+
+    def test_too_many_decimal_spaces_generator_should_return_decimal_with_more_decimal_places_than_declared(self):
+        declared_decimal_places = 2
+        result = generate_too_many_decimal_places(declared_decimal_places)
+        decimal_places = len(str(result).split('.')[1])
+        self.assertEqual(decimal_places, 3)
```

Comment:
You didn't fix tests with new functions.

Reply:
![image](https://user-images.githubusercontent.com/17569113/52334651-7e5e8100-2a00-11e9-941a-4ca4ef1085a3.png)

Reply:
Done